DXMainClient/Domain/Multiplayer/CnCNet/MapSharer.cs

```diff
@@ -78,8 +76,14 @@ private static void Upload(object mapAndGame)
 
             Logger.Log("MapSharer: Starting upload of " + map.BaseFilePath);
 
-            bool success = false;
-            string message = MapUpload(MAPDB_URL, map, myGameId, out success);
+            if (string.IsNullOrWhiteSpace(ClientConfiguration.Instance.CnCNetMapDBUploadURL))
+            {
+                Logger.Log("MapSharer: Upload URL is not set.");
```

Comment:
```suggestion
                Logger.Log("MapSharer: Upload URL is not configured.");
```
Maybe it would also be good to show the note about stuff not being configured to the user?